database/buildSchema/16_template_element.sql

```diff
@@ -6,7 +6,11 @@ CREATE TYPE public.template_element_category AS ENUM (
 
 CREATE TYPE public.is_reviewable_status AS ENUM (
     'ALWAYS',
-    'NEVER'
+    'NEVER',
+    'OPTIONAL_IF_NO_RESPONSE'
+    -- TO-DO:
+    -- 'OPTIONAL_IF_RESPONSE',
+    -- 'OPTIONAL' (the above two combined)
```

Comment:
I would have liked to do these other optional variations (i.e. allow review to NOT review a question that HAS been answered), but one thing at a time...